templates/includes/form.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+
+  {% block h1 %}
+  {% endblock %}
+
+  <form action="" method="post" novalidate>

```

Comment:
The `novalidate` attribute in the form tag disables HTML5 form validation. Ensure this is intentional, as it might lead to forms being submitted without client-side validation. If server-side validation is solely relied upon, this is fine, but it's generally good to have both client-side and server-side validation.